android/src/main/java/com/middlewarereactnative/MiddlewareReactNativeModule.java

```diff
@@ -103,7 +103,8 @@ public void nativeCrash() {
     new Thread(() -> {
       try {
         Thread.sleep(2000);
-      } catch (InterruptedException e) {}
+      } catch (InterruptedException e) {
+      }
       throw new RuntimeException("test crash");
```

Comment:
Is it possible to throw exception fromm catch block itself.

Reply:
Throwing exception might lead sdk to crash causing customer app to crash.